src/tools/dbimport/Main.cpp

```diff
@@ -103,8 +103,15 @@ bool StartDB()
 {
     MySQL::Library_Init();
 
-    // Load databases
-    DatabaseLoader loader("dbimport");
+    // Load modules conditionally based on what modules are allowed to auto-update or none
+    std::string modules = sConfigMgr->GetOption("Updates.AllowedModules", std::string(""));
```

Comment:
I don't think that the casting to `std::string` is necessary here, you could just put `""`

Reply:
Moreover, you should set `GetOption<std::string>`

Reply:
Thanks for feedback, I am not a C++ dev as I do full stack web dev and devops as my day job.

Will make the change and commit back.